lib/javascript.js

```diff
@@ -71,8 +71,14 @@ module.exports = {
     'jsdoc/check-line-alignment': 'error',
     'jsdoc/check-syntax': 'error',
     'jsdoc/multiline-blocks': 'error',
-    'jsdoc/newline-after-description': 'error',
     'jsdoc/require-hyphen-before-param-description': ['error', 'never'],
+    'jsdoc/tag-lines': [
+      'error',
+      'any',
+      {
+        startLines: 1,
+      },
+    ],
```

Comment:
This is a replacement of removed `'jsdoc/newline-after-description': 'error',`. see https://github.com/gajus/eslint-plugin-jsdoc/releases/tag/v42.0.0